source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResultClient/SettlementReportDataFactory.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Globalization;
+using Energinet.DataHub.Wholesale.CalculationResults.Infrastructure.CalculationResultClient.Mappers;
+using Energinet.DataHub.Wholesale.CalculationResults.Interfaces.CalculationResultClient;
+using NodaTime.Text;
+
+namespace Energinet.DataHub.Wholesale.CalculationResults.Infrastructure.CalculationResultClient;
+
+public static class SettlementReportDataFactory
+{
+    public static IEnumerable<SettlementReportResultRow> Create(Table table)
```

Comment:
Just a thought - what do you think?
```suggestion
public static class SettlementReportFactory
{
    public static SettlementReport Create(Table table)
```

Reply:
Not sure. I guess it depends on if we we will handle results and basis data as separate calls. I'm not sure which direction we will go in here, so I just kept it "low-level".